chat/tests/test_chat_consumer.py

```diff
@@ -52,8 +59,118 @@ def create_user() -> User:
                         'username': user.username,
                         'message': message}
 
-    # Testing Leaving the room
+    # Getting list of users in the room
+    await communicator.send_json_to({'command': 'room_users', 'room': room.id})
+    response = await communicator.receive_json_from()
+    last_update: float = response['data']['users']['1']['last_update']
+    assert isinstance(last_update, float)
+    assert response == {
+        'data': {
+            'users': {
+                '1': {
+                    'name': 'Alireza Savand',
+                    'username': 'alireza',
+                    'left': False,
+                    'last_update': last_update
```

Comment:
Last update can be:

* Last time messaged
* Last time did any kind of activities
* etc

For now, it means last message or last time joined